lib/fortune.js

```diff
@@ -252,6 +253,14 @@ Fortune.prototype._exposeResourceDefinitions = function() {
   });
 };
 
+Fortune.prototype.customType = function(name, schema, options, schemaCallback) {
```

Comment:
The schema here is the thing passed into mongoose, right? It should also accept userland schema to expose in `/resources` output.